esp/clients/wsdfuaccess/wsdfuaccess.cpp

```diff
@@ -525,6 +525,8 @@ StringBuffer &encodeDFUFileMeta(StringBuffer &metaInfoBlob, IPropertyTree *metaI
 
         const char *keyPairName = metaInfo->queryProp("keyPairName");
         const char *privateKeyFName = environment->getPrivateKeyPath(keyPairName);
+        if (isEmptyString(privateKeyFName))
+            throw makeStringExceptionV(-1, "Key name %s is not found in environment settings: /EnvSettings/Keys/KeyPair.", keyPairName);
```

Comment:
trivial: perhaps change to '%s' - it can sometimes be confusing depending on naming if unquoted.

Reply:
@jake I just made the change.